jsonnet/custom.libsonnet

```
@@ -8,9 +8,13 @@
             alert: 'etcdGRPCRequestsSlow',
             expr: |||
               histogram_quantile(0.99, sum(rate(grpc_server_handling_seconds_bucket{job="etcd", grpc_method!="Defragment", grpc_type="unary"}[10m])) without(grpc_type))
-              > 1
+              > on () group_left (type)
+              bottomk(1,
+                0.5 * group by (type) (cluster_infrastructure_provider{type="Azure"})
+                or
+                0.25 * group by (type) (cluster_infrastructure_provider))
```

```suggestion
              bottomk(1,
                1.5 * group by (type) (cluster_infrastructure_provider{type="Azure"})
                or
                1 * group by (type) (cluster_infrastructure_provider))
```
I think is what you'd want if we think that 1.5s p99 is right for Azure and 1 otherwise. I haven't studied CI jobs on this one though to see what values yield > 95% pass rate.